### PR TITLE
Automate release workflow via CircleCI

### PR DESCRIPTION
**Pull request by ashb**

### CODE REVIEW

**Review by andriisoldatenko — APPROVED** [review]

**Comment by andriisoldatenko** [comment]
@ashb what about `flake8`?

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+            python -m venv .venv
+            . .venv/bin/activate
+            pip install -e '.[test]'
+      - save_cache:
+          key: deps-{{ .Branch }}-{{ checksum "setup.py" }}
+          paths:
+            - ".venv"
+            - "~/.cache"
+      - run:
+          name: run_tests
```
Comment (andriisoldatenko) [inline .circleci/config.yml:57]: what about to run flake8 as it was in drone?

Comment (andriisoldatenko) [reply .circleci/config.yml:57]:
```
    commands:
    - pip install flake8 'flake8-import-order>=0.18'
    - flake8
```

Comment (ashb) [reply .circleci/config.yml:57]: Oh yes, good spot.

(I should add that to my other new plugin too.)

Comment (ashb) [reply .circleci/config.yml:57]: Added

**Pull request merged**